docker-compose.yml

```diff
@@ -31,6 +31,7 @@ services:
       # doesn't match the platform under which Docker Engine is running. e.g.
       # WSL, Docker for Mac, etc.
       - /var/app/apps/fz_http/assets/node_modules
+      - /var/app/_build
```

Comment:
Hmm, don't think it's necessary? It's not mounted.

Reply:
For me it was helping to prevent recompiling modules for the test env each time I ran... could tweak as necessary though.

Reply:
Ah cool. I keep a container around for tests so didn't think about this. That's pretty useful.